hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -557,8 +557,9 @@ public static Schema getLatestTableSchema(String path, org.apache.hadoop.conf.Co
     try {
       HoodieTableMetaClient metaClient = StreamerUtil.createMetaClient(path, hadoopConf);
       return getTableAvroSchema(metaClient, false);
-    } catch (Exception e) {
-      LOG.warn("Error while resolving the latest table schema", e);
+    } catch (Throwable throwable) {
+      LOG.warn("Error while resolving the latest table schema.", throwable);
+      // ignored
     }
```

Comment:
it's not a good practice to ignore Throwable. why don't you want to fail loud here?

Reply:
Just to consistent with `org.apache.hudi.table.catalog.HoodieCatalog#getLatestTableSchema` which catch `Throwable` rather than `Exception`, why only ignore `Exception`, there are many situation that not the subclass of Exception for example `oom`.

Reply:
@TJX2014 then `getLatestTableSchema` should be fixed too. We don't catch Throwable because Error should not be caught. Quote from javadoc

> An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch

So what is the strong reason to catch and ignore errors like OOM? You'd need to fail loud in that case.